sdk/identity/azure-identity/azure/identity/_internal/get_token_mixin.py

```diff
@@ -66,8 +66,6 @@ def get_token(
         :keyword str claims: additional claims required in the token, such as those returned in a resource provider's
             claims challenge following an authorization failure.
         :keyword str tenant_id: optional tenant to include in the token request.
-        :keyword bool enable_cae: indicates whether to enable Continuous Access Evaluation (CAE) for the requested
```

Comment:
What's the reasoning for removing the enable_cae docstrings? I see that it creates a cascade of apiview changes.

Reply:
Because in our implementation, we only pass enable_cae when it is True for the sake of backward compatibility and mypy is not 100% happy if we have it.

Reply:
The mypy check fails if we add `enable_cae` to the signature here (I.e. `enable_cae: bool = False`)? What is mypy specifically complaining about? I feel like it should be agnostic to what we pass it from core.

Reply:
Currently, our code is

return self._credential.get_token(*scopes, claims=claims, tenant_id=tenant_id, **kwargs)

if enable_cae is in the signature, we would need to change it to 

return self._credential.get_token(*scopes, claims=claims, tenant_id=tenant_id, enable_cae=enable_cae, **kwargs)

because it is not in kwargs anymore.

As we discussed, we don't want to always pass enable_cae into get_token, we only specify it when it is True.

Reply:
> As we discussed, we don't want to always pass enable_cae into get_token, we only specify it when it is True.

Wasn't this just from the perspective of the core auth policies calling `get_token`? Functionally, enable_cae = False is equivalent to it being None. I'd hate to lose the documentation for this arg.

Also, I remember in the past, we had a [PR](https://github.com/Azure/azure-sdk-for-python/pull/31047) to match the signature of the TokenCredential protocol [`get_token`](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/core/azure-core/azure/core/credentials.py#L22). Seems like not including `enable_cae` will be straying from this pattern.